test/test_mbjson.py

```diff
@@ -317,6 +321,34 @@ def test_recording_instrument_credits(self):
         self.assertEqual(m['performer:acoustic guitar'], 'Ed Sheeran')
 
 
+class RecordingMultiArtistsTest1(MBJSONTest):
+    """Test multiple artists with some common contries.
+    """
+
+    filename = 'recording_multi_artists_1.json'
+
+    def test_recording_multi_artists_1(self):
+        m = Metadata()
+        t = Track('1')
+        recording_to_metadata(self.json_doc, m, t)
+        self.assertEqual(m['~artistcountry'], 'GB & US')
+        self.assertEqual(m['~artists_countries'], 'GB; US; US')
+
+
+class RecordingMultiArtistsTest2(MBJSONTest):
+    """Test multiple artists with one unknown (missing) country.
+    """
+
+    filename = 'recording_multi_artists_2.json'
+
+    def test_recording_multi_artists_2(self):
+        m = Metadata()
+        t = Track('1')
+        recording_to_metadata(self.json_doc, m, t)
+        self.assertEqual(m['~artistcountry'], 'GB, US & XX')
```

Comment:
I find it weird we use different formats for both lists, what about using a comma + whitespace only?

```python
self.assertEqual(m['~artistcountry'], 'GB, US,  XX')
self.assertEqual(m['~artists_countries'], 'GB, US, XX')
```

Naming isn't very clear, one needs to check documentation to understand the difference between both.

Reply:
> I find it weird we use different formats for both lists, what about using a comma + whitespace only?

I'm okay with comma and whitespace only for separators.  That will actually simplify the code a bit.

> Naming isn't very clear, one needs to check documentation to understand the difference between both.

I actually tried to kind of follow the existing naming structure ("albumartist" and "albumartistsort" for the strings, and "\_albumartists" and "\_albumartists_sort" for the multi-values), but I'm open to alternate suggestions, especially if it will make it clearer for the users.

Reply:
Thinking about this a bit more, we can probably get by with only providing two multi-value variables (**\_albumartist_countries** and **\_artist_countries**), each containing the full list of country codes (including duplicates) for all credited artists.  The user can then use the **$unique()** function to get a consolidated list if they want to get rid of the duplicates.

Reply:
Removed the unique **\_albumartistcountry** and **\_artistcountry** variables in [cfff376](https://github.com/metabrainz/picard/pull/2627/commits/cfff37667d2b4c5beceff1d1d6f3471b3b35ab21).  This leaves only **\_albumartists_countries** and **\_artists_countries** multi-value variables, which contain the full artists country lists.  These can easily be converted to unique lists using the **$unique()** scripting function.